src/app/setup/github-oauth.component.html

```diff
@@ -90,40 +104,55 @@
           </div>
         </div>
       </div>
+    </div>
 
-      <div class="step-three">
-        <div class="col-xs-12">
-          <div class="col-xs-1">
-            <div class="circle pull-right"><p>3</p></div>
-          </div>
-          <div class="col-xs-11">
-            <h2>Configure Integration Location</h2>
-            <div>Specify where to store the integration.</div>
-            <div *ngIf="!stepThreeComplete">
-              <form class="form-horizontal"
-                    role="form">
-                <div class="form-group">
-                  <label for="connectedAccount"
-                         class="col-xs-4 control-label">Connected Account</label>
-                  <div class="col-xs-8">
-                    <select id="connectedAccount"
-                            class="form-control"
-                            *ngIf="noAccountConnected">
-                      <option value="">Choose...</option>
-                      <option value="">Option 1</option>
-                      <option value="">Option 2</option>
-                      <option value="">Option 3</option>
-                    </select>
-                    <i *ngIf="!noAccountConnected">No account connected.</i>
-                  </div>
-                </div>
-              </form>
+
+    <!-- Step Three -->
+
+    <div class="step-three"
+         [ngClass]="{'step-complete' : stepThreeComplete === true}">
+      <div class="col-xs-10">
+        <div class="col-xs-1">
+          <div class="circle pull-right"><p>3</p></div>
+        </div>
+        <div class="col-xs-11">
+          <h2>Configure Integration Location</h2>
```

Comment:
I guess this isn't `configure` any more but rather `this is where integrations will be stored` :-)

Reply:
I think we're doing away with this step altogether so should be okay! :)

Reply:
Yep, might as well delete the commented out stuff really.